gateway/grpc/README.md

```diff
@@ -69,7 +79,7 @@ Verify external loadbalancing by transmitting 10 RPCs over one channel.  The res
 
 ```bash
 $ docker run --add-host grpc.domain.com:$GW_XLB_VIP  \
-  -t gcr.io/cloud-solutions-images/grpc_app /grpc_client \
+  -t docker.io/salrashid123/grpc_app /grpc_client \
```

Comment:
I understand gcr.io/cloud-solutions-images have been deprecated. But i'm not moving these to a personal docker account is a good move here neither. Due to rate limiting as you mentioned @salrashid123 and also what happens if you or whoever host these images in their docker account leaves Google and closes the account !

@mark-church wdyt ?

Reply:
thats fair (though the docker.io image is already in the repo [here](https://github.com/GoogleCloudPlatform/gke-networking-recipes/blob/master/ingress/single-cluster/ingress-custom-grpc-health-check/grpc-hc-proxy.yaml#L47) which i'll change here as well)

i'm fine with the image reference here hosted on a more stable repo (does  a mantainer here have push rights to`gcr.io/google-samples/*?  if so then feel free to build and push from the HC proxy and app here

* https://github.com/salrashid123/grpc_health_proxy/tree/master/example
* https://github.com/salrashid123/grpc_health_proxy/blob/master/Dockerfile

Reply:
Let me try to figure that one out. The test app we use here (whereami) is also on gcr.io/google-samples so that sounds like a good place for your proxy.

Reply:
@salrashid123 In order to move the code to gcr.io/google-samples the code itself have be under https://github.com/GoogleCloudPlatform/kubernetes-engine-samples which owned by the DevRel team.

I'm happy to fork your repo and move it under the GKE samples repo if you are ok with that ?

Reply:
thats fine.  
should i keep this PR pending till then and update the final image url

Reply:
Yeap i will let you know once the image is in google-samples and we can merge this

Reply:
I discussed with mark. We decided to use your docker file from docker.io. Somewhere this year gRPC HC's will be supported in GKE and we will not need the proxy anymore